db/models/item.test.js

```diff
@@ -0,0 +1,152 @@
+'use strict'
+
+const db = require('APP/db'),
+  { Item } = db,
+  { expect } = require('chai'),
+  Promise = require('bluebird')
+
+describe('The `Item` model', () => {
+  /**
+   * First we clear the database and recreate the tables before beginning a run
+   */
+  before('Await database sync', () => db.didSync)
+
+  /**
+   * Next, we create an (un-saved!) item instance before every spec
+   */
+  let item
+  beforeEach(() => {
+    item = Item.build({
+      price: 1000.00,
+      quantity: 3
+    })
+  })
+
+  /**
+   * Also, we empty the tables after each spec
+   */
+  afterEach(() => {
+    return Promise.all([
+      Item.truncate({ cascade: true })
+    ])
+  })
+
+  describe('attributes definition', function() {
```

Comment:
You can item.save() once here and then test the attributes off of that saved instance.